### PR TITLE
i#1979: print module id + offset for ret addr in drltrace

### LINKED ISSUE #1979: Print module id + offset for return address in drltrace
We have to print a module id + offset for return addresses in drltrace instead of an absolute address which we use now and provide a list of modules at the end of execution to translate the module ids to paths.

### PR DESCRIPTION
**Pull request by mxmssh**
Implemented module id + offset printing scheme instead of
raw address for return addresses in drltrace.

Fixes #1979

### CODE REVIEW

File: drltrace/drltrace.c [hunk 1]
```diff
@@ -275,13 +279,15 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
...
-     */
-    dr_fprintf(outf,
-               options.print_ret_addr ? " and return to " PFX: "",
-               drwrap_get_retaddr(wrapcxt));
+    if (options.print_ret_addr) {
+        ret_addr = drwrap_get_retaddr(wrapcxt);
+        res = drmodtrack_lookup(drcontext, ret_addr, &mod_id, &mod_start);
+        if (res == DRCOVLIB_SUCCESS) {
+            dr_fprintf(outf,
+                       options.print_ret_addr ? " and return to module id:%d, offset:" PIFX: "",
```
Comment (derekbruening) [inline drltrace/drltrace.c]: suggest: space before : part of ?: to make it easier to read

Comment (mxmssh) [inline drltrace/drltrace.c]: Do you mean space after PIFX ?

Comment (derekbruening) [inline drltrace/drltrace.c]: Yes, I found it hard to parse as written.
